doc/nrf/getting_started.rst

```diff
@@ -18,6 +18,7 @@ After that, pick a sample that is related to the application you want to create
    gs_ins_os
    gs_assistant
    gs_installing
+   gs_recommended_versions
```

Comment:
This should go after gs_ins_os (thus before the installation instructions).

Reply:
fixed.